tests/test-runner/bin/zts-report.py

```diff
@@ -239,6 +239,9 @@
     'userquota/setup': ['SKIP', exec_reason],
     'vdev_zaps/vdev_zaps_004_pos': ['FAIL', '6935'],
     'zvol/zvol_ENOSPC/zvol_ENOSPC_001_pos': ['FAIL', '5848'],
+    'pam/setup': ['SKIP', "pamtester might be not available"],
+    'pam/pam_basic': ['SKIP', "pamtester might be not available"],
+    'pam/pam_nounmount': ['SKIP', "pamtester might be not available"]
```

Comment:
As long as it's checked in `pam/setup` we don't need exceptions for all of the tests since they will never be run.